### ISSUE TITLE
Cannot change Linked Node of photos

### ISSUE THREAD

**Issue by quaestor** [theCrag/website#3936]
The details page of photos used to show a button that allowed to change the Linked Node of a photo. This button is gone, so the only way to move a picture between nodes now is to delete it and upload it at the right node.

**Comment by scd** [comment]
It should still be there. You have to click edit first

**Comment by quaestor** [comment]
It‘s not there:
![F1DE7E46-F22A-493F-A3B1-DDEF74F72C91](https://user-images.githubusercontent.com/54371/123747605-70e91080-d8b3-11eb-9ddf-13e8536e40de.png)
![1029E396-AF5E-447E-90CD-E2C96A550EF4](https://user-images.githubusercontent.com/54371/123747613-734b6a80-d8b3-11eb-9dab-97e9778b2cdf.png)

**Comment by scd** [comment]
What is the url of the photo. This is probably a permissions issue. Is this a photo you uploaded?

**Comment by quaestor** [comment]
https://www.thecrag.com/photo/4951772832

It‘s a photo I uploaded.

**Comment by scd** [comment]
I think it is because it is a webcover. Remove the webcover then see what happens.

**Comment by quaestor** [comment]
You‘re right! It works for photos that aren‘t webcovers. Maybe this should be documented better? Or display the button but redirect to a page that asks if you would like to remove the cover?

**Comment by scd** [comment]
Leave the issue open and change the description for better documentation

**Comment by quaestor** [comment]
Ok, changed the description. Sorry for the hassle.

But while we're at it: I would prefer the UI to be the same independent of the (webcover) state of the photo. Say you're uploading a photo, then somebody else sets it as a webcover. Then suddenly you cannot delete the photo or link it to another crag, which is confusing.
